### ISSUE TITLE
WYSIWYG - REPL Demo needed. Tool that provides instant visual feedback preview of the output.

### ISSUE THREAD

**Issue by sebacampos** [raycharius/slack-block-builder#52]
**Is your feature request related to a problem? Please describe.**

There's no way to build **visually** and live preview the output of what you're building unless you run the code... 
There are the [Build Methods](https://www.blockbuilder.dev/#/build-methods) but you need to run code, take the output of that, and go to Slack Block Kit Builder... or get the link via `printPreviewUrl()` and use that...
I think would be nice to have a  WYSIWYG feature.

**Describe the solution you'd like**
A WYSIWYG REPL tool,  similar to [https://jsx-slack.netlify.app/](https://jsx-slack.netlify.app/) ?

**Describe alternatives you've considered**

**Additional context**

**Comment by raycharius** [comment]
Thanks for pushing this change! This is definitely something that I want to get to, but it will require some R&D to see how feasible it is going to be to host it on the [website](https://www.blockbuilder.dev). It's built on [docsify](https://github.com/docsifyjs/docsify/), and I'm not sure yet if it's possible to create such a page and host it there. 



I'm a bit busy with a few other projects and finishing up Version 2, but if you want to collaborate on this and have the time to contribute, we can talk next steps!

**Comment by zaini** [comment]
Created a demo of this here: http://old.zaini.me/slack-block-builder-viewer/

Is this sort of what you want? I definitely agree that something like this would be useful but I think the JSX syntax lends to an online editor more.

This demo will output the result of running .buildToJSON() on the given block. It will also include a link to the Slack Block Builder with the blocks inserted so you can view the design.

I could add variables but for now, I think this should get the idea across. I haven't looked into how jsx-slack deals with variables (if at all)

Here are some features and things that could be done to improve this (because currently it sucks lol):

* Nicer code editor and design for the actual site
* Embed the result onto the page instead of having to click a button
* create tests and actually make it something useable

**Comment by raycharius** [comment]
@zaini – nice! Are you a part of the Slack Community space? If so, and if you'd be up for collaborating on getting this production ready, would love to chat about how we can get this up and running!

**Comment by zaini** [comment]
Yeah would be interested collaborating on this. I'm not currently part of any Slack community spaces but please share wherever it would be best to chat about this.
